input/write_reactors.py

```diff
@@ -125,8 +196,11 @@ def input_render(reactor_file, region_file, template, output_file):
         reactor = fp.read()
     with open(region_file, 'r') as bae:
         region = bae.read()
+    startmonth = (init_date // 100) % 100
```

Comment:
you could re-use the new get_ymd(yyyymmdd) function here instead of copy and pasting.